framework/test/unit/modules/token/init_genesis_state_fixture.ts

```diff
@@ -135,22 +135,6 @@ export const invalidGenesisAssets = [
 		},
 		"tokenID' maxLength exceeded",
 	],
-	[
```

Comment:
same as above

Reply:
This test checks `initGenesis`, which decode a Buffer into object, and expects a "uint64 overflow".

However, since it's impossible to encode a value that exceeds uint64 limit. I think it is safe to remove this check and let the unit test above to handle the test

Reply:
i see it's failing. technically it's possible to have more value than uint64 if someone uses different encoder, but i guess it's fine to leave it out